src/pages/Forum/ForumPage.css

```
@@ -571,7 +572,511 @@
+  align-items: flex-start;
+  gap: 6px;
+}
+
+/* Bubble */
+.cm-bubble {
+  display: inline-block;
+  padding: 9px 13px;
+  background: var(--bg-secondary, #f0f2f5);
+  border-radius: 4px 18px 18px 18px;
+  max-width: 100%;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)